src/main/scala/com/ibm/couchdb/api/Documents.scala

```diff
@@ -30,40 +30,40 @@ class Documents(client: Client, db: String, typeMapping: TypeMapping) {
   val types  = typeMapping.types
   val server = new Server(client)
 
-  private def getClassName[T](obj: T): String = obj.getClass.getCanonicalName
+  private def getClassName[S](obj: S): String = obj.getClass.getCanonicalName
```

Comment:
with upickle 0.3.x this seems to create a type parameter name conflict, with the following error message "type parameter T defined in method x shadows type T defined in method ...  You may want to rename your type parameter, or possibly remove it.".  The simplest work around was to rename the type parameter